src/shared/widgets/web-contents/iframe-mozbrowser.jsx

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2016 Mozilla
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+this file except in compliance with the License. You may obtain a copy of the
+License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed
+under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+*/
+
+import React, { PureComponent } from 'react';
+
+import WebContents from '.';
+
+export default class IframeMozBrowser extends PureComponent {
+  constructor(...args) { // eslint-disable-line
+    super(...args);
+  }
+
+  componentDidMount() {
+    this._iframe.addEventListener('mozbrowserloadstart', () => {
+      this.props.onDidStartLoading();
+    });
+    this._iframe.addEventListener('mozbrowserloadend', () => {
+      this.props.onDidStopLoading();
+      this.props.onPageTitleSet({ title: 'Loaded' });
```

Comment:
The reason we set the title to 'Loaded' in unprivileged web contexts is because of cors: we can't really get much information out from the iframe, and tab titles are not accessible. I'd imagine in the case of qbrt and mozbrowser, we can do a lot better here?